packages/slate-editor/src/icons/Component-TikTok.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <rect width="32" height="32" rx="4.8" fill="#374151" />
+    <path fill-rule="evenodd" clip-rule="evenodd"
```

Comment:
_Comment in an unrelated place_

Hey @e1himself, there's a problem that I see with SVGs which I'm not sure how to solve.

So we're using an SVG optimizer that minifies the classes and IDs for SVG parts, this will result in a conflicting situation with some of the SVGs sharing the same id for their clip-path URL with the same ID, see here:

Giphy:
![image](https://github.com/prezly/slate/assets/67554982/4d76bfaa-08d5-4cd3-9256-852c5c04d094)

Vimeo:
![image](https://github.com/prezly/slate/assets/67554982/b9684bda-25c2-4e1e-b80b-44caff7401ad)

Do you have any solution in mind?

Reply:
"Some, not all" points to the original SVGs (and I can confirm that these have clip paths when the others don't). Let me try and make you some that don't have these clip paths.

An aside, but I think that maybe we put new on the 'embed' section menu, instead of on every embed? think on every one looks a bit overwhelming (I know you won't see this when filtered however?)

Reply:
OK, a little more research:

1. SVGR is using SVGO under the hood.
2. SVGO is using [`cleanupIds` plugin](https://github.com/svg/svgo/blob/main/plugins/cleanupIds.js) to minify and cleanup the ids.
3. We need a way to tell SVGR to disable that plugin.

But I'm not sure how to do it in our build system. I am using the web GUI here: https://jakearchibald.github.io/svgomg/

And if you disable the "Clean up IDs" plugin, it won't try cleaning up the IDs.

Reply:
>  (I know you won't see this when filtered however?)

That's true, because when filtered, there is only 1 SVG with the clip path ID of `#a` for example.

> Let me try and make you some that don't have these clip paths.

But this is a problem with our build system in its current form, we should be able to solve it correctly to avoid this stuff from happening in the future. However I can accept that as a middle ground solution :-)

Reply:
@MohammadxAli unless your tool is automated, and you can't control the output, I'd change the reference ald link to reference to something unique i.e. something like this will make it unique compared to the other SVG.
![CleanShot 2023-08-24 at 14 49 26@2x](https://github.com/prezly/slate/assets/858646/c5ca8580-fa9c-4648-9452-13fd60b47c8d)

Reply:
Well, that's the problem, the SVGs I have added to the codebase already use a unique ID.

But our build system which uses SVGR/SVGO turns those unique IDs into non-unique IDs :')

`blah-blah-blah` -> `a`
`meh-meh-meh` -> `a`

BOOM.

Reply:
Here is the complete list of SVGs that use clip-path which potentially will result in conflicting IDs:

- Component-GoogleDocs
- Component-GoogleSheets
- Component-Tally
- Component-Giphy
- Component-Vimeo

cc @alpower, can you somehow remove the clip-path from those in Figma so that I can re-export them?

Reply:
Fixed the source Figma file so no clip paths exist. ✅

Reply:
Hey :wave: 

Good job figuring out how to work it around :trophy: 

> But I'm not sure how to do it in our build system. I am using the web GUI here: https://jakearchibald.github.io/svgomg/
> And if you disable the "Clean up IDs" plugin, it won't try cleaning up the IDs, which I expect to solve the issue.

Just for future reference, I believe there is a way to configure this:
- The Babel transformation plugin we use for SVGs:
  https://github.com/birdofpreyru/babel-preset-svgr#under-the-hood
- SVGR configuration reference:  
  https://react-svgr.com/docs/options/#svgo
- svgo.config.js reference:
  https://github.com/svg/svgo#configuration

But just to be clear: let's not spend time on fixing the configuration now :)